bigquery_etl/glean_usage/common.py

```diff
@@ -150,7 +150,9 @@ def get_app_info():
     app_info = {}
 
     for app in apps_json:
-        if app["app_name"] not in app_info:
+        if app["app_id"].startswith("rally-"):
```

Comment:
```suggestion
        if app["app_id"].startswith("rally"):
```

This was missing `rally.debug`.